server/services/discovery/enrichment/cli_utils.py

```
@@ -78,9 +91,9 @@ def run_cli_command(cmd, env=None, timeout=DEFAULT_TIMEOUT):
         )
         if result.returncode != 0:
             stderr = result.stderr.strip()
-            return None, f"Command failed (rc={result.returncode}): {stderr}"
+            return None, f"[{label}] Command failed (rc={result.returncode}): {stderr}"
         return result.stdout.strip(), None
     except subprocess.TimeoutExpired:
-        return None, f"Command timed out after {timeout}s: {' '.join(cmd)}"
+        return None, f"[{label}] Command timed out after {timeout}s"
     except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)